satpy/enhancements/__init__.py

```diff
@@ -48,3 +48,23 @@ def cira_stretch(img, **kwargs):
         chn -= np.log10(0.0223)
         chn /= 1.0 - np.log10(0.0223)
         chn /= 0.75
+
+
+def lookup(img, **kwargs):
```

Comment:
Could you add support for 1- and 4-channels images ? And maybe add a small unit test ?